lib/route-recognizer.js

```diff
@@ -89,13 +89,13 @@ EpsilonSegment.prototype = {
   generate: function() { return ""; }
 };
 
-function parse(route, names, specificity) {
+function parse(allSegments, route, names, specificity) {
   // normalize route as not starting with a "/". Recognition will
   // also normalize.
   if (route.charAt(0) === "/") { route = route.substr(1); }
 
   var segments = route.split("/");
-  var results = new Array(segments.length);
+  //var results = new Array(segments.length);
```

Comment:
what % of improvement was this over  https://github.com/tildeio/route-recognizer/pull/95/files#r63742200